src/main/resources/templates/jiraIssueModule.vm

```diff
@@ -4,6 +4,18 @@ $webResourceManager.requireResource("de.uhd.ifi.se.decision.management.jira:jira
 <div id="treant-container">
 	<p>No Element has been selected</p>
 </div>
+<script>
+var dialog = document.getElementById("dialog");
+if (dialog) {
```

Comment:
This prevents the dialog from beeing included more than once. Is there a better alternative to do this?